### PR TITLE
Add Docker Engine installation instructions for Ubuntu

### PR DESCRIPTION
**Pull request by jakolehm**

### CODE REVIEW

**Comment by SpComb** [comment]
> `$ echo "deb http://dl.bintray.com/kontena/kontena /" | sudo tee -a /etc/apt/sources.list`

Could this be simplified to:

````
$ echo "deb http://dl.bintray.com/kontena/kontena /" > /etc/apt/sources.list.d/kontena.list
```

**Comment by SpComb** [comment]
Also on Ubuntu xenial:

```
W: GPG error: http://dl.bintray.com/kontena/kontena  Release: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 379CE192D401AB61
E: The repository 'http://dl.bintray.com/kontena/kontena  Release' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

Need to either sign the bintray `Release` with a GPG key, or use `apt update --allow-insecure-repositories`

**Comment by jakolehm** [comment]
@SpComb did you run `$ wget -qO - https://bintray.com/user/downloadSubjectPublicKey?username=bintray | sudo apt-key add -` ?

File: docs/getting-started/installing/ubuntu.md [hunk 1]
```diff
@@ -10,8 +10,20 @@ title: Ubuntu
...
+Kontena requires [Docker Engine](https://docs.docker.com/engine/) to be installed on every host (master and nodes). Preferred method for installing Docker Engine is to use Docker's APT repositories:
+
+```
+$ sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+$ echo "deb https://apt.dockerproject.org/repo ubuntu-trusty main" | sudo tee -a /etc/apt/sources.list.d/docker.list
+$ sudo apt-get update
+$ sudo apt-get install apt-transport-https ca-certificates linux-image-extra-$(uname -r) linux-image-extra-virtual
+$ sudo apt-get install docker-engine=1.11.2-0~trusty
+$ sudo apt-mark hold docker-engine
+```
```
Comment (SpComb) [inline docs/getting-started/installing/ubuntu.md]: The older `docker.io/xenial=1.10*` packages are not supported.

**Pull request merged**